extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -201,6 +209,31 @@ class AmpStickyAd extends AMP.BaseElement {
       this.viewport_.updatePaddingBottom(0);
     });
   }
+
+  /**
+   * To check for background-color alpha and force it to be 1.
+   * Whoever calls this needs to make sure it's in a vsync.
+   * @private
+   */
+  forceOpacity_() {
+    if (!isExperimentOn(this.win, UX_EXPERIMENT)) {
+      return;
+    }
+
+    // TODO(@zhouyx): Move the opacity style to CSS after remove experiments
+    // Not using setStyles because we will remove this line later.
```

Comment:
? I will remove once we change our ux and decide to remove experimental protection.